README.md

```diff
@@ -49,7 +49,7 @@ The Terraform Provider for RedFish is released and licensed under the MPL-2.0 li
 
 | **Terraform Provider** | **iDRAC Version** | **OS** | **Terraform** | **Golang** |
 |---------------------|-----------------------|-------|--------------------|--------------------------|
-| v1.1.0 | 5.x <br> 6.x <br> 7.x | ubuntu22.04 <br> rhel8.x <br> rhel9.x | 1.4.x <br> 1.5.x <br> | 1.20
+| v1.1.0 | 5.x <br> 6.x <br> 7.x | ubuntu22.04 <br> rhel9.x | 1.5.x <br> 1.6.x | 1.21
```

Comment:
version needs to be updated to 1.2.0

Reply:
The 2nd column header should be iDrac9 Firmware Version.